python/ray/dataframe/dataframe.py

```diff
@@ -154,7 +176,7 @@ def shape(self):
         """
         return (len(self.index), len(self.columns))
 
-    def _map_partitions(self, func, *args):
+    def _map_partitions(self, func, index=None):
```

Comment:
Nothing in `_map_partitions` was using `*args`, so I removed it. We do need to sometimes give a new Index, though.